src/components/Logo.jsx

```diff
@@ -0,0 +1,7 @@
+import React from "react";
+
+function Logo({ width = "100px" }) {
+  return <div>Logo</div>;
```

Comment:
send height also to prevent the dynamic shift in the re-paint ( after asset fetch )

Reply:
Adedd height